src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyOutput = 0;
+        int buyOutput = 0;
+        int result = 0;
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
The BufferedReader instance should be a local variable within the try-with-resources block.